dm/tests/_utils/run_tidb_server

```diff
@@ -5,20 +5,32 @@
 
 set -eu
 
+tmp_config="/tmp/dm_test/tidb.toml"
+
 PORT=$1
 PASSWORD=$2
 CONFIG=""
+
 if [ "$#" -ge 3 ]; then
-	CONFIG=$3
+	cat $3 >$tmp_config
+else
+	# turn on collation framework https://docs.pingcap.com/tidb/stable/character-set-and-collation#new-framework-for-collations
+	rm $tmp_config || true
+	cat >$tmp_config <<EOF
+new_collations_enabled_on_first_bootstrap = true
```

Comment:
why not let cases which depends on this flag opens it separately?

Reply:
change the default test behaviour to avoid we forget collation compatibility in future (such as a new developer add a test and reviews forget checking collation).